py/torch_tensorrt/dynamo/lowering/passes/accumulate_fp32_matmul.py

```diff
@@ -1,49 +1,60 @@
 import logging
 
 import torch
+from torch_tensorrt.dynamo._settings import CompilationSettings
 from torch_tensorrt.dynamo.lowering.passes.pass_utils import (
     clean_up_graph_after_modifications,
 )
 
 logger = logging.getLogger(__name__)
 
 
-def accumulate_fp32_matmul(gm: torch.fx.GraphModule) -> torch.fx.GraphModule:
+def accumulate_fp32_matmul(
+    gm: torch.fx.GraphModule, settings: CompilationSettings
+) -> torch.fx.GraphModule:
     """Replace a matmul layer with fp32 accumulation nodes"""
-    matmul_targets = [
-        torch.ops.aten.mm.default,
-        torch.ops.aten.bmm.default,
-        torch.ops.aten.addmm.default,
-    ]
-    matmul_nodes = [node for node in gm.graph.nodes if node.target in matmul_targets]
-    for matmul_node in matmul_nodes:
-        # Prior to the matmul node, insert a cast to the 32-bit float32 node
-        node_inputs = matmul_node.all_input_nodes
-
-        for node_input in node_inputs:
-            with gm.graph.inserting_before(matmul_node):
-                node_32bit = gm.graph.call_function(
+    if settings.use_fp32_acc:
```

Comment:
just return the original graph module if not use_fp32_acc (maybe write a debug message)

Reply:
done